examples/plot_autoreject_workflow.py

```diff
@@ -41,7 +41,11 @@
 dataset = 'ds002778'  # The id code on OpenNeuro for this example dataset
 subject_id = 'pd14'
 
-target_dir = op.join(op.dirname(autoreject.__file__), '..', 'examples')
+target_dir = os.path.join(
+    os.path.dirname(autoreject.__file__), '..', 'examples', dataset)
+if not os.path.isdir(target_dir):
```

Comment:
openneuro doesn't make it if it doesn't exist?

Reply:
it takes the `dataset` parameter as a dir name and creates it if it doesn't exist, but only if no `target_dir` argument is passed: https://github.com/hoechenberger/openneuro-py/blob/c6f994cebe67c78e3208f34ac4e7d8753fce506a/openneuro/_download.py#L696-L699

Reply:
I simplified it a bit in https://github.com/autoreject/autoreject/pull/303/commits/eba75a73724f8eccd34c4d47a32d13ff0acef1cb